src/classes/TDTM_Config_API_Test.cls

```diff
@@ -92,5 +92,31 @@ private class TDTM_Config_API_Test {
         con = [SELECT AccountId FROM Contact WHERE Id = :con.Id LIMIT 1];
         system.assertEquals(null, con.AccountId, 'there should not be an account id associated with this new contact');
     }
-    
+
+
+    /*******************************************************************************************************
+    * @description verify the TDTM Callable Action does not modify the TDTM Triggers globally
+    ********************************************************************************************************/
+    @IsTest
+    private static void triggersAreDisabledForCurrentContext() {
+        // skip the test if Advancement is installed
+        if (ADV_PackageInfo_SVC.useAdv()) {
+            return;
+        }
+
+        // Get list of TDTM modified dates
+        Integer accountCountBefore = Database.countQuery('SELECT COUNT() FROM Account');
+
+        TDTM_Config_Api.disableAllTriggers();
```

Comment:
@patrick-yan-sf One more question. Should this method call be `CallableApi.run('TDTM.DisableAllTriggers', null)` so that the test can validate the entire process?

Reply:
I'm not sure.   It makes sense that we want to run the entire process.  But putting CallableApi in to the TDTM_Config_Api_Test would mean the TDTM class references the CallableApi class.  And the CallableApi class already references the TDTM class.  So there would be a circular type of reference.  I think we talked about this yesterday.  We might want to look at an API test suite?  Something that just test the global APIs that we provide.

Reply:
Actually this is just the TEST class that references the CallableApi class.  So maybe it's ok.  I can add it if you think that is fine.

Reply:
This is the test method that currently calls TDTM_Config_Api directly. We could either change this method to call the CallableApi class, or change the `CallableApi_Test` class to validate that the triggers are disabled by calling the run method with `TDTM.DisableAllTriggers`. Writing this out, I think it makes sense to add this validation logic to the `CallableApi_Test` class rather than this class.

Reply:
I've added the check on CallableApi_Test to make sure the triggers were indeed disabled